src/sagemaker/tensorflow/estimator.py

```diff
@@ -229,7 +229,7 @@ def __init__(self, training_steps=None, evaluation_steps=None, checkpoint_path=N
             **kwargs: Additional kwargs passed to the Framework constructor.
         """
         if framework_version is None:
-            LOGGER.warning(fw.empty_framework_version_warning(TF_VERSION, TF_VERSION))
+            LOGGER.warning(fw.empty_framework_version_warning(TF_VERSION, self.LATEST_VERSION))
```

Comment:
self.LATEST_VERSION looks wrong to me as LATEST_VERSION is not class variable, it is defined outside the __init__. 
It should be just LATEST_VERSION

Reply:
it's defined in the class `TensorFlow`: https://github.com/aws/sagemaker-python-sdk/blob/42c7f308e417d945680e200d3e80743e43ab50d3/src/sagemaker/tensorflow/estimator.py#L170-L174

`LATEST_VERSION` (without `self`) would work only if it is defined outside of the `TensorFlow` class:
```python
>>> class A:
...     FOO = 1
...     def __init__(self):
...         print(FOO)
... 
>>> A()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 4, in __init__
NameError: name 'FOO' is not defined
```

Reply:
Ok Thanks for the explanation.